packages/calcite-components/src/components/dropdown/dropdown.e2e.ts

```diff
@@ -1148,35 +1147,53 @@ describe("calcite-dropdown", () => {
       ).toBe(false);
     });
 
-    it("dropdown wrapper should have height when filter results empty and combined with a PickList in Panel  #3048", async () => {
+    it("Dropdown wrapper should not extend the height of the Picklist filter input when filter results are empty in Panel  #3048", async () => {
```

Comment:
This test was failing but after a conversation with @anveshmekala I updated the test to check for the original use case. Dropdown wrapper now only has a height when the Dropdown is open.